app/javascript/publishers/home.js

```diff
@@ -183,6 +183,11 @@ document.addEventListener('DOMContentLoaded', function() {
         pendingContactEmail.innerText = updatedEmail;
         showPendingContactEmail(updatedEmail);
 
+        let currentUserName = document.getElementById('current-user-name');
+        let userNameDropDown = document.getElementById('user-name-dropdown');
```

Comment:
Generally I suggest using classes of the style `js-user-name-dropdown`. When reading the templates this helps you identify where JS might rely on the DOM. Especially for un-tested behavior it can help prevent regressions.

Reply:
Will change that over to a class and prefix with `js-`. By the way, do you prefer adding parent scope `.user-menu` into the selector?

`let userNameDropDown = document.querySelector('.user-menu .js-user-name-dropdown');`

Reply:
Un-scoped class selectors are faster, and I don't think the scope adds much so I would leave it out 👍